Example/ChatExample/Screens/CommentsExampleView.swift

```diff
@@ -24,7 +24,7 @@ enum Action: MessageMenuAction {
         }
     }
     
-    func icon() -> Image {
+    func icon(theme:ChatTheme.Images) -> Image {
```

Comment:
I see why you'd want this change, I also considered this while adding themes, but I just don't see how to use it conveniently in your own MessageMenuAction. since MessageMenuAction takes Image - I think the user can pass his own asset together with color and everything if need be